tutorials/concepts/potential-problems-with-contracts.md

```diff
@@ -28,14 +28,16 @@ can issue warnings from being ever written, but that would require
 Marlowe contracts to be dependently-typed, and writing expressions that
 are dependently-typed is much more cumbersome.
 
-Instead, Marlowe allows contracts that issue warning to be written, and
-we provide [static analysis tools](static-analysis.md) that let contract
-developers check whether a particular contract can possibly issue
-warnings. Additionally, we provide fall-back behaviours for when a
-contract produces a warning, despite our advice. We provide fall-back
-behaviours because we acknowledge that analysing big contracts can be
-very computationally expensive, and because mistakes can be made. We
-want badly written contracts to fail in the most harmless way possible,
+### Static analysis
+
+Marlowe allows contracts that issue warning to be written, and
+we provide [static analysis tools](/tutorials/guides/static-analysis) 
+that let contract developers check whether a particular contract can
+possibly issue warnings. Additionally, we provide fall-back behaviours
+for when a contract produces a warning, despite our advice. We provide
+fall-back behaviours because we acknowledge that analysing big contracts
+can be very computationally expensive, and because mistakes can be made.
+We want badly written contracts to fail in the most harmless way possible,
```

Comment:
The spelling of words should use U.S. standards. This slips through the cracks since so much content was initially written in the U.K. But let's replace every instance of "behaviours" with "behaviors" and "analysing" with "analyzing." 

Also, at the end of the paragraph: "in the most harmless way possible, that is conservatively." Let's add a comma before the last word. 

"that is, conservatively."